sdk/keyvault/azure-security-keyvault-keys/test/samples/sample3-get-keys/sample3_get_keys.cpp

```diff
@@ -38,8 +38,15 @@ int main()
   auto credential
       = std::make_shared<Azure::Identity::ClientSecretCredential>(tenantId, clientId, clientSecret);
 
-  KeyClient keyClient(std::getenv("AZURE_KEYVAULT_URL"), credential);
-
+  KeyClient client(std::getenv("AZURE_KEYVAULT_URL"), credential);
+  try
+  {
+    KeyVaultKey key = client.GetKey("some_key").Value;
+  }
+  catch (Azure::Core::RequestFailedException ex)
```

Comment:
```suggestion
  catch (const Azure::Core::RequestFailedException& ex)
```

Always catch by const reference. It is rarely needed to catch by non-const (if you need to modify the exception), and it is even more rare that you would need to catch by value, which creates a copy when an exception is caught. I won't be surprised if we would never need to catch anything other than a const ref throughout the entire life of Azure SDK.